mlflow/pipelines/steps/train.py

```diff
@@ -134,14 +134,14 @@ def _validate_and_apply_step_config(self):
                 error_code=INVALID_PARAMETER_VALUE,
             )
         self.skip_data_profiling = self.step_config.get("skip_data_profiling", False)
-        if "estimator_method" not in self.step_config:
+        if "estimator_method" not in self.step_config and self.step_config["using"] in [
+            "estimator_spec"
+        ]:
             raise MlflowException(
                 "Missing 'estimator_method' configuration in the train step.",
                 error_code=INVALID_PARAMETER_VALUE,
             )
```

Comment:
Can we update the exception message to mention using estimator_spec?

Reply:
Done